src/shared-components/accordion/__snapshots__/test.tsx.snap

```diff
@@ -64,7 +64,7 @@ exports[`<Accordion /> renders disabled accordion 1`] = `
 
 .emotion-5:focus {
   outline: none;
-  box-shadow: inset 0px 0px 0px 2px #332E54;
+  box-shadow: inset 0px 0px 0px 3px #332E54;
```

Comment:
I dont like this change visually, waiting for Ben review

Reply:
Yeah, I wonder if this change was unintentional, since I just worked on these last cycle.

Reply:
In referencing the focusInner it should be 2px. Figma denotes 3px to account for the border we apply

Reply:
changed both themes to 2px